app/scripts/components/data-catalog/index.tsx

```diff
@@ -47,7 +48,8 @@ import {
   TAXONOMY_TOPICS
 } from '$utils/veda-data';
 import { DatasetClassification } from '$components/common/dataset-classification';
-import { allDatasets } from '$components/exploration/data-utils';
+
+const allDatasetsForCatalog = Object.values(datasets).map((d) => d!.data);
```

Comment:
💭  : It would be nice to have this at a global level in the same place & way we would access `datasets` globally as well in the future. Maybe expose this filtered arr from a state / method store too

Reply:
It turns out it already exists: https://github.com/NASA-IMPACT/veda-ui/blob/main/app/scripts/utils/veda-data.ts#L17 :[  It is my fault that I didn't look through more carefully, but also it is quite confusing that our data utils related to veda are spread over. I will try to consolidate after this sprint.